.circleci/mysql-rebuild/docker-compose.yml

```diff
@@ -13,8 +13,6 @@ services:
       MYSQL_DATABASE: circle
       MYSQL_HOST: mysql
       # Copy vars from host to the container.
-      # Acquia Cloud - v1
-      AC_API_KEY:  $AC_API_KEY
```

Comment:
This key would be for a deprecated version of the API that doesn't appear to be in use anywhere.